### PR TITLE
Form validation

### PR DESCRIPTION
**Pull request by virag-ky**
We added an event listener to the form element, when the user wants to submit the form, first it validates that the email is in lower case only. If it is, then the form can be submitted, if not, then the form can not be submitted and the user gets an error message under the email input element, telling the user to use a valid email.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -237,6 +241,7 @@ <h2>
             placeholder="Email"
             required
           />
+          <small class="small hidden show">Please, enter valid Email: lower case only.</small>
```
Comment (thecodechaser) [inline index.html:244]: - [ ] Great job working on the functionality but there is a little issue. You are supposed to show an error message to the user near the submit button but right now it's after the input field. Please fix it and make sure you show the error message near the submit button.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## Hi @virag-ky 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good accessibility report. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by virag-ky** [comment]
I changed the position of the error message, near the submit button.

**Review by gemmen29 — APPROVED** [review]
Hi @virag-ky, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**